images/common/install-sambacc-common.sh

```diff
@@ -30,27 +30,44 @@ install_sambacc() {
     done
 
 
-    local action=install-from-copr-repo
-    if [ "${#wheels[@]}" -gt 1 ]; then
-        echo "more than one wheel file found"
-        exit 1
-    elif [ "${#wheels[@]}" -eq 1 ]; then
-        action=install-wheel
+    local action="" 
+    echo "INFO: determining sambacc installation source." 
+    if [ "${#wheels[@]}" -gt 0 ]; then
+        echo "INFO: wheel found"
+        if [ "${#wheels[@]}" -gt 1 ]; then
+            echo "ERROR: more than one wheel file found"
+            exit 1
+        elif [ "${#wheels[@]}" -eq 1  ]; then
+            action=install-wheel
+        fi
     fi
-
-    if [ "${#rpmfiles[@]}" -gt 1 ]; then
-        echo "more than one sambacc rpm file found"
-        exit 1
-    elif [ "${#rpmfiles[@]}" -eq 1 ]; then
-        action=install-rpm
+    if [ "${#rpmfiles[@]}" -gt 0 ]; then
+        echo "INFO: rpm file found"
+        if [ "${#rpmfiles[@]}" -gt 1 ]; then
+            echo "ERROR: more than one sambacc rpm file found"
+            exit 1
+        elif [ "${#rpmfiles[@]}" -eq 1 ]; then
+            action=install-rpm
+        fi
     fi
-
-    if [ "${#repofiles[@]}" -gt 1 ]; then
-        echo "more than one repo file found"
-        exit 1
-    elif [ "${#repofiles[@]}" -eq 1 ]; then
-        action=install-from-repo
+    if [ "${#repofiles[@]}" -gt 0 ]; then
+       echo  "INFO: repo file found"
+       if [ "${#repofiles[@]}" -gt 1 ]; then
+           echo "ERROR: more than one repo file found"
+           exit 1
+       elif [ "${#repofiles[@]}" -eq 1 ]; then
+           action=install-from-repo
+       fi
     fi
+       if [ -z "${action}" ]; then
+
+           echo "INFO: no local sambacc installation source found. falling back to copr install."
+           action=install-from-copr-repo
+       fi
```

Comment:
Looking a lot better now, but this section is still overly indented.

Reply:
really fixed now